distribution/dashboard/ui/src/components/customModal/OperateModal.tsx

```diff
@@ -54,5 +51,13 @@ export default function OperateModal({
     return <UpgradeTenantModal {...props} />;
   }
 
+  if(type === 'modifyUnitSpecification'){
+    return <ModifyUnitDetailModal {...props} />
+  }
+
+  if(type === 'changeUnitCount'){
+    return <ModifyUnitModal {...props} />;
```

Comment:
The unit quantity is an integer. `InputNumber` may be more appropriate than `Input` for the internal input box.

Reply:
Ok.